tests/ragger/README.md

```diff
@@ -0,0 +1,64 @@
+# Ragger functional tests
+
+> :point_right: Every path on this document assumes you are at the root of the
+repository.
+
+These tests are implemented in Python with the `Ragger` library, allowing to
+write tests which will run on both the
+[Speculos](https://github.com/LedgerHQ/speculos) emulator and physical devices
+by using either [LedgerComm](https://github.com/LedgerHQ/ledgercomm) or
+[LedgerWallet](https://github.com/LedgerHQ/ledgerctl) libraries.
+
+
+## Dependencies
+
+Python dependencies are listed in [requirements.txt](requirements.txt), install
+them using [pip](https://pypi.org/project/pip/)
+
+```
+pip install --extra-index-url https://test.pypi.org/simple/ -r requirements.txt
+```
+
+The extra index allows to fetch the latest version of Ragger.
+
+
+## Launching the tests
+
+### On Speculos
+
+Given the requirements are installed, you need to generate the application binaries
+for all the SDK. The tests will expect the binaries to be:
+
+- `elfs/boilerplate_nanos.elf`
+- `elfs/boilerplate_nanox.elf`
+- `elfs/boilerplate_nanosp.elf`
+
+Once that's done, just do:
+
+```
+pytest tests/ragger
+```
+
+This will run the tests inside the Speculos emulator, for all the current SDK
+versions (NanoS, NanoX and NanoS+).
```

Comment:
Not really true? It will only test for the SDK used to build the elfs files?

Reply:
Indeed, confusing sentence